summoner-cli/src/Summoner/CLI.hs

```diff
@@ -392,6 +394,12 @@ githubP d = flag Idk d
          <> short 'g'
          <> help "GitHub integration"
 
+ghActionsP :: Decision -> Parser Decision
+ghActionsP d = flag Idk d
+             $ long "actions"
+         <> short 'a'
```

Comment:
I don’t use the pure-CLI interface, so I don’t have strong feelings about what UI y’all want around this. Happy to take suggestions as to what is desirable, though `-a` doesn’t seem to be occupied at the moment.